string prop_name
float64 angular_velocity